README.md

```diff
@@ -290,8 +290,8 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
 
                 // The path to the TLS certificates directory used to establish
                 // secure connections between the HTTP API and the OpenID provider.
-                // Typically, this should point to a standard certificate directory
-                // such as "/etc/ssl/cert".
+                // This should not be set unless your TLS certificates directory is
+                // not a standard certificate directory such as "/etc/ssl/cert".
```

Comment:
Consider rewording this. It is a little hard to comprehend due to multiple uses of the word "not".

Try stating when this property SHOULD be used rather than when it shouldn't.

Reply:
> Please update to be closer to the wording @trel provided.

Will update depending on what is decided here: https://github.com/irods/irods_client_http_api/pull/444#discussion_r2254286786

Reply:
Updated the wording.